docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
-/>
+<DocsVideo src="https://vimeo.com/806005654" />
 
-For users of the open-source [Cypress app](/guides/core-concepts/cypress-app),
-we've integrated test run information from Cypress Cloud so developers can see
-the latest results across the team, and identify areas of concern.
+### Analyze and diagnose test health
 
-<DocsImage
-  src="/img/dashboard/v10/runs-list-in-cypress-app.png"
-  alt="Runs List"
-/>
+View each [run's overview](/guides/cloud/runs#Overview-tab) to see rich
+analytics and compare past runs and analyze trends over time. You can quickly
+analyze changes in your setup that might introduce problematic trends as well as
+identify unreliable tests with
+[Flaky Test Management](/guides/cloud/flaky-test-management).
 
-### Run tests in parallel, in priority order, or not at all
+<DocsVideo src="https://vimeo.com/806900039" />
 
-With our [Smart Orchestration](/guides/cloud/smart-orchestration) features, you
-can run tests across a swarm of machines simultaneously while Cypress Cloud
-coordinates runners and
-[balances test loads](/guides/guides/parallelization#Balance-strategy) - no
-setup required! You can prioritize recently failed specs with
-[Spec Prioritization](/guides/cloud/smart-orchestration#Spec-Prioritization) to
-surface problems earlier, and cancel whole test runs on failure with
-[Auto Cancellation](/guides/cloud/smart-orchestration#Auto-Cancellation) to save
-on resource usage. You can also
-[cancel in-progress runs](/guides/cloud/runs#Cancel-run) manually from Cypress
-Cloud if you need to.
+### Reduce test time & CI costs
+
+[Smart Orchestration](/guides/cloud/smart-orchestration) enables you to run
+tests across multiple machines simultaneously in your CI environment while
+Cypress Cloud coordinates runners and
+[balances test loads](/guides/guides/parallelization#Balance-strategy)!
 
 <DocsImage
   src="/img/dashboard/introduction/orchestration-diagram.png"
   alt="Diagram comparing serial and parallel test configurations"
 />
 
-### Integrate with source control providers
+Cypress Cloud's
+[Spec Prioritization](/guides/cloud/smart-orchestration#Spec-Prioritization)
+lets you prioritize recently failed specs so they run faster on their next run.
+The [Auto Cancellation](/guides/cloud/smart-orchestration#Auto-Cancellation)
+feature will cancel a whole test run on failure, saving time and resource usage.
+You can also [cancel in-progress runs](/guides/cloud/runs#Run-cancellation)
+manually from Cypress Cloud if you need to.
+
+### Integrate with your favorite tools
+
+#### Source Control providers
 
-Ensure rock-solid reliability by keeping failing code out of your
-[GitHub](https://github.com), [GitLab](https://gitlab.com) and
-[Bitbucket](https://bitbucket.org) repositories, with
-[status checks](/guides/cloud/integrations/github#Status-checks) that block
-commits from being merged until your Cypress tests are green. Surface test
+Cypress Cloud integrates with GitHub, GitLab, and Bitbucket to provide
```

Comment:
Weird that we linked to the providers directly before, but we could link to the integration guides here for reference.